greenpeace/planet4-master-theme
# PLANET-7390: Update to Timber 2.1

Ref: https://jira.greenpeace.org/browse/PLANET-7390
Requires: https://github.com/greenpeace/planet4-base/pull/295
Doc: [Timber upgrade guide](https://timber.github.io/docs/v2/upgrade-guides/2.0/)

A configured dev env is available on branch [planet4-develop@feat/timber-2](https://github.com/greenpeace/planet4-develop/tree/feat/timber-2)

## Test

Check all templates :grimacing: :
- author page
- navigation
- search
- single post
- single page


**How to test this PR on local dev env?**
- Please checkout 'feat/timber-2' [branch](https://github.com/greenpeace/planet4-develop/tree/feat/timber-2) on local dev env root(planet4-develop) -
- You need to update your local with
`npm run env:install`
- This will also checkout 'feat/timber-2' branch on master-theme repo.
(This might override local changes so please take backup or use a spare local env for this.)

If any warnings are triggered due to early translation loading or similar issues, you can temporarily deactivate the WP-Stateless plugin locally using the following command:
`npx wp-env run cli wp plugin deactivate wp-stateless`

Also, please ensure that the timber-library plugin is not active.
If it's active, deactivate it using the command below:
`npx wp-env run cli wp plugin deactivate timber-library`
(This plugin conflicts with the Composer-based Timber 2.0 installation.)
